### ISSUE TITLE
The accuracy of SCNN

### ISSUE THREAD

**Issue by xskyz** [zwx8981/DBCNN-PyTorch#5]
Hello, thanks for your code !
Can you tell me the accuracy of SCNN's classification of 39 distortion types? If I want to retrain SCNN, what is the appropriate accuracy( I can stop my training when I get it)?

**Comment by zwx8981** [comment]
@xskyz Hello, you may get accuracy around 97%~98%.

**Comment by xskyz** [comment]
@zwx8981 Now I used the pre-trained SCNN model (copying parameters other than the classifier) ​​to train the live dataset, but only got about 84%. Is it because the amount of data is too small? I have not done any preprocessing on the data.

**Comment by zwx8981** [comment]
@xskyz What do you want the SCNN to do? Classifying distortion types of images in LIVE dataset?

**Comment by xskyz** [comment]
@zwx8981 Yes, I want to classify the five distortion type in LIVE dataset.

**Comment by zwx8981** [comment]
@xskyz You may try freezing all parameters of the pre-trained SCNN and only fine-tune your newly-initialized FC layer for the distortion types classification.  You may also use SCNN to extract off-the-shelf features and train a SVM for classification.

**Comment by xskyz** [comment]
@zwx8981 OK, I will try your suggestion, thank you for your guidance~

**Comment by xskyz** [comment]
@zwx8981 I can only get 85% when I only fine-tune my newly-initialized FC layer. I want to use neural networks to classify, but I don't know how to improve its accuracy.

**Comment by zwx8981** [comment]
@xskyz Did you try different hyper-parameters? (learning rate, batch size etc.)

**Comment by xskyz** [comment]
@zwx8981 Yes, I have tried.But it did not get better results.

**Comment by zwx8981** [comment]
@xskyz Sounds strange, did you train and test on all 982 images of LIVE or only on distorted images (779 in total)?

**Comment by xskyz** [comment]
@zwx8981 oh!!!  I train and test on all 982 images.

**Comment by zwx8981** [comment]
@xskyz OK, you should only use distorted images~
